dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/SKChatMessage.cs

```diff
@@ -10,11 +10,21 @@ namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.AzureSdk;
 public class SKChatMessage : ChatMessageBase
```

Comment:
Yeah, I'm not sure about that. Maybe some ambiguity between SK ChatMessage entity and ChatMessage from Azure SDK.

Reply:
@lemillermicrosoft This was a concrete implementation of our `ChatMessageBase` abstraction. So this is not a SDK class, but an extension specialization from our Abstraction, `SK` in the name as @dmytrostruk pointed out is to avoid the conflict with the `ChatMessage` class specific to the SDK.

Reply:
I'd argue the private `ChatMessage` should be `SKChatMessage` and the extension be `ChatMessage` as there is nothing really SK about it, is there? It's in the AzureSdk namespace, which makes me wonder if it should be named AzureChatMessage?

@RogerBarreto  -- is there a reason why `ChatMessage` is private and not public?